src/components/Popover/Popover.tsx

```diff
@@ -79,8 +85,13 @@ export const Popover: React.FunctionComponent<PopoverProps> & {
 
     const firstFocusable = findFirstFocusableNode(activatorContainer.current);
     const focusableActivator = firstFocusable || activatorContainer.current;
-    setActivatorAttributes(focusableActivator, {id, active, ariaHaspopup});
-  }, [active, ariaHaspopup, id]);
+    setActivatorAttributes(focusableActivator, {
```

Comment:
Rather than adding an `activatorDisabled` prop, what do you think about deriving the value rather than explicitly setting it?

```ts
// HTMLElement doesn't contain `disabled` in its interface
// We could alternatively use instanceof `focusableActivator instnaceof HTMLButton && focusableActivator.disabled`
 const activatorDisabled = (focusableActivator as HTMLButtonElement).disabled;
```

Reply:
The reason I ended up going the prop route is that the `Popover` can't expect/know what kind of element its activator is or whether it's even focusable. The above snippet would work, but we'd need to chain all other possibilities to that conditional (it could be a textfield, etc). Since disabling a `Button`, `TextField` etc is done explicitly, just telling the `Popover` that the activator, whatever it may be, is disabled explicitly is a lot easier to test.

Reply:
I wasn't aware that activators can be non-focusable. Maybe this is something we should try and enforce through types. Seems like a huge accessibility issue to not be able to open a popover through keyboard interactions 😬 cc/ @dpersing since your assigned as well, what do you think?

Using `HTMLButtonElement` type was just a quick example. There's other ways we can achieve the same type safety that'll work on any disablable element e.g.
```tsx
    const focusableActivator: HTMLElement & {disabled?: boolean} =
      firstFocusable || activatorContainer.current;
    const activatorDisabled =
      'disabled' in focusableActivator && Boolean(focusableActivator.disabled);
```

Reply:
Unfortunately adding the typing alone doesn't help. The real source of the problem is the `findFirstFocusableNode` utility because it [eliminates `disabled` elements as focusable selectors](https://github.com/Shopify/javascript-utilities/blob/b59e7fb82da0cf96aeb93f234763932be6af39e0/src/focus.ts#L3). 

Modifying the `FOCUSABLE_SELECTOR`s to remove the `:not(disabled)` in `@shopify/javascript-utilities` could cause unexpected breaking changes for other consumers of the utility, so I've updated the PR to use our own version.